Code/Framework/AzTest/AzTest/aztest_files.cmake

```diff
@@ -7,15 +7,23 @@
 #
 
 set(FILES
-    AzTest.h
     AzTest.cpp
+    AzTest.h
     ColorizedOutput.cpp
+    GemTestEnvironment.cpp
+    GemTestEnvironment.h
+    MockComponentApplication.cpp
```

Comment:
AzTest is used by many libraries and does not include mocks that are specific to them. What is the argument for AzCore mocks living inside AzTest? Should they not exist in a separate sister library?